src/api/api_core.php

```diff
@@ -137,6 +150,7 @@ function logout() {
 
         setcookie('auth_key', '', time()-3600, '/');
         setcookie('auth_userid', '', time()-3600, '/');
+        setcookie('expiry_time', '', time()-3600, '/');
```

Comment:
Yeah, i definitely think that you should rename the `expiry_time` cookie and give it a value of true or false somehow (as strings would be fine) --- There are two completely distinct comparisons going on, right:
* comparisons involving the cookie named `expiry_time`, which we compare against a value we specify for the purpose of setting a user-defined boolean, and we can do that any way we like because it's our code
* comparisons involving the parameter `$expiryTime`, which nominally are in some fashion time-based, and are set by browser interpretations of the cookie protocol implementation.  This we're trying to reverse-engineer enough to make it do what we want

Naming our cookie `expiry_time` is confusing, because it conflates those two things, one which we have control over and one which we don't.  So we should name our cookie something else, and make its value a boolean (or string representation of a boolean, whatever works more straightforwardly) because that's what it is.